pyad/forward_diff.py

```diff
@@ -98,6 +159,36 @@ def __rpow__(self, other):
 
 
 class Variable(Tensor):
+    '''
+    Variable (a variable value for the purposes of differentiation). A variable is used to
+    represent any value which the user wishes to include in the partial derivative outputs
+    and is built on top of the Tensor base class.
+
+    As a variable a name parameter is required (example: x, y or z).
+
+    Parameters
+    ----------
+    name: str
+        Store the name of the variable
+
+    value : np.array
+        Store the value of the variable
+
+    d : MultivariateDerivative
```

Comment:
This actually isn't one of the parameters of the init. Maybe we could move this description down to the attributes section